src/CreateGuesser.tsx

```diff
@@ -31,9 +32,27 @@ const displayOverrideCode = (schema, fields) => {
   console.info(code);
 };
 
+interface IntrospectedCreateGuesserProps extends CreateProps {
+  children: any;
+  fields: any;
+  initialValues: any;
+  margin: any;
+  readableFields: any;
+  redirect: string;
+  sanitizeEmptyValues: any;
+  schema: any;
+  schemaAnalyzer: any;
+  simpleFormComponent: any;
+  submitOnEnter: any;
+  successMessage: any;
+  toolbar: any;
+  validate: any;
+  variant: any;
+  warnWhenUnsavedChanges: any;
+  writableFields: any;
+}
+
 export const IntrospectedCreateGuesser = ({
-  fields,
-  readableFields,
```

Comment:
Are you sure it's a good idea to remove them? We probably want to filter them.

Reply:
They are not used (`'fields' is defined but never used. eslint@typescript-eslint/no-unused-vars`)

Do you prefer that i put it back and modify the eslint rules or leave as it ? (same for the other comments)

Reply:
React-admin is using a `sanitizeRestProps` function, maybe we can use something like this as well?